sdk/python/lib/pulumi/output.py

```diff
@@ -256,7 +256,7 @@ def from_input(val: Input[T]) -> 'Output[T]':
             # Since Output.all works on lists early, serialize the class's __dict__ into a list of lists first.
             # Once we have a output of the list of properties, we can use an apply to re-hydrate it back as an instance.
             items = [[k, Output.from_input(v)] for k, v in val.__dict__.items()]
-            fn = cast(Callable[[List[Any]], T], lambda props: typ(**{k: v for k, v in props}))  # type: ignore
+            fn = cast(Callable[[List[Any]], T], lambda props: typ(**props))  # type: ignore
```

Comment:
This change breaks the `Output.from_input` tests. https://github.com/pulumi/pulumi/pull/6674/checks?check_run_id=2241937838#step:17:2096

Might need to add `# pylint: disable=unnecessary-comprehension` like line 268

Reply:
Done.